app.json

```diff
@@ -1,20 +1,18 @@
 {
   "expo": {
+    "name": "insulince",
+    "slug": "insulince",
+    "description": "Insulince is a project aiming to help anyone with Insulin Resistance Syndrome to track nutrition and food intake. https://github.com/rmurphy178/insulince",
+    "version": "0.1.0",
     "sdkVersion": "19.0.0",
     "ios": {
-      "bundleIdentifier": "insulince"
+      "bundleIdentifier": "com.app.insulince"
```

Comment:
bundle identifier and android package use reverse-dns notation so they need to be in this format https://en.wikipedia.org/wiki/Reverse_domain_name_notation